packages/pages/package.json

```diff
@@ -57,6 +57,7 @@
     "THIRD-PARTY-NOTICES"
   ],
   "dependencies": {
+    "@types/prompts": "^2.4.0",
```

Comment:
Does this need to be a dep? You already have it as a devDep. You shouldn't need it in both places at the very least.